desc.en.hocon

```diff
@@ -4703,7 +4703,7 @@
       "label": "License string"
     },
     "license_root": {
-      "desc": "Defines the EMQX Enterprise license.\n\n\nThe default license has 100 connections limit, it is issued on 2023-01-09 and valid for 5 years (1825 days).\n\nEMQX comes with a default trial license.  For production use, please\nvisit https://www.emqx.com/apply-licenses/emqx to apply.",
+      "desc": "Defines the EMQX Enterprise license.\n\n\nThe default license has 25 connections limit, it is issued on 2023-12-08 and valid for 5 years (1825 days).\n\nEMQX comes with a default trial license.  For production use, please\nvisit https://www.emqx.com/apply-licenses/emqx?version=5 to apply.",
```

Comment:
```suggestion
      "desc": "Defines the EMQX Enterprise license.\n\n\nThe default license has a limit of 25 connections. It was issued on 2023-12-08 and valid for 5 years (1825 days).\n\nEMQX comes with a default trial license.  For production use, please\nvisit https://www.emqx.com/apply-licenses/emqx?version=5 to apply.",
```

Reply:
Please do not change English version docs here.
the latest doc for 5.4 is here: https://github.com/emqx/emqx/blob/5ad46ce077deb0222440fd4598cfcd43285d1b6f/rel/i18n/emqx_license_schema.hocon#L42-L53